examples/column_view_datagrid/grid_cell/grid_cell.ui

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <template class="GridCell" parent="GtkBox">
```

Comment:
Please, avoid subclassing from complicated widgets like `GtkBox` and such, especially if you don't even use the extra features like in this case.

The correct pattern to create a custom widget in GTK4 is to subclass from `GtkWidget` and use one a layout manager. A `GtkBinLayout` is enough in this case. You can set the layout manager in the `class_init` function via `klass.set_layout_manager_type::<GtkBinLayout>()`. Also, don't forget to dispose the child in the `dispose` function.

Reply:
@bilelmoussaoui was faster here.

Reply:
nice, I changed it to subclass GtkWidget instead now, and added the dispose